BaseFilter/sections/specific.txt

```diff
@@ -853,6 +853,7 @@ techarmor.xyz##.my-ad-container
 !
 ! NOTE: Specific rules
 !
+dustebin.com#%#//scriptlet('prevent-addEventListener', '/^(?:click|touchend)$/', 'window[_0x')
```

Comment:
I wouldn't add this as the website has low traffic.

Reply:
i do see low traffic but i added because of ad server,if you want i can remove site specific filter,let adserver in block